Survey/Configurations/Plists/GoogleService/Production/GoogleService-Info.plist

```diff
@@ -3,21 +3,21 @@
 <plist version="1.0">
 <dict>
 	<key>CLIENT_ID</key>
-	<string>859927641745-pn5hpdeqsog4mnospccftgn7a089pf2l.apps.googleusercontent.com</string>
+	<string>1042384710612-5mum9u65bbafqkb38fpv7ckmart15ra2.apps.googleusercontent.com</string>
```

Comment:
Optional: A great way to hide Firebase keys is to encode it to [base64](https://docs.github.com/en/actions/security-guides/encrypted-secrets#storing-base64-binary-blobs-as-secrets) and store them in Github Secrets, then you can use them later by [decoding it on CICD](https://github.com/suho/kmm-ic/blob/develop/.github/workflows/deploy_ios_staging_build.yml#L43)

Reply:
@blyscuit Thank you for the suggestion, I will note this 🙌 .